.github/workflows/composite/maximize-build-space/action.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: maximize-build-space
```

Comment:
Nitpick: We could include a short rationale why we don't use an existing action for this (as far as I'm concerned the reason is that this one does less and is simpler)

Reply:
Done